ci/pipeline.yml

```diff
@@ -1,79 +1,94 @@
 resource_types:
-- name: s3-iam
-  type: docker-image
-  source:
-    repository: 18fgsa/s3-resource
-- name: slack-notification
-  type: docker-image
+
+- name: registry-image
+  type: registry-image
   source:
-    repository: cfcommunity/slack-notification-resource
-- name: bosh-deployment
```

Comment:
The `bosh-deployment` resource needs to be re-added. It looks like there isn't a hardened image for this one yet, but @dandersonsw would know better than I

Reply:
I put this back as requested.